tracer/test/Datadog.Trace.TestHelpers/SpanMetadataV1Rules.cs

```diff
@@ -173,6 +173,7 @@ public static Result IsCosmosDbV1(this MockSpan span) => Result.FromSpan(span)
                 .IsOptional("db.name")
                 .Matches("db.type", "cosmosdb")
                 .IsPresent("out.host")
+                .IsPresent("out.port")
                 .IsPresent("peer.service")
```

Comment:
`peer.service` and `_dd.peer.service.source` were added in a previous commit, but we don't run CosmosDb integration tests in CI so this did not cause test failures before

Reply:
Should we add it to v0 as well?

Reply:
Mmm I don't think so. In v0 the `out.host` tag contains the full URI (including the host) so it may be confusing to the user to have an additional `out.port` tag